src/main/java/com/woowacamp/soolsool/core/member/domain/Member.java

```diff
@@ -94,6 +95,10 @@ public void update(final MemberModifyRequest memberModifyRequest) {
         this.address = new MemberAddress(memberModifyRequest.getAddress());
     }
 
+    public void updateMileage(final String amount) {
+        this.mileage = this.mileage.charge(new BigInteger(amount));
+    }
```

Comment:
```suggestion
    public void updateMileage(final MemberMileage amount) {
        this.mileage = this.mileage.charge(amount);
    }
```

동일한 클래스끼리 메시지를 주고받는게 적절할 것 같아요.

Reply:
MemberMileage 객체를 생성하는 것은 Service 보단 Member domain의 역할이지 않을까 싶은데 어떻게 생각해??

Reply:
사용자의 요청으로 `MemberMileage`를 생성하는 것도 비즈니스 로직으로 보여서 Service가 적절한 것 같아dy.

Reply:
흠.. 이 부분은 각자 스타일이 조금 다른것 같고 나는 개인적으로 둘 다 좋은 방향이라고 생각해 종하랑 재휘 의견 들어보고 넷이 통일하는 것도 좋은 방법이지 싶어~

Reply:
회의록에 작성하고 저녁먹고 이야기해보입시더!